index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Alex Corey</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href="style.css" rel="stylesheet" type="text/css" media="screen" href="main.css" />
+    <!-- <script src="main.js"></script> -->
+</head>
+
+<body>
+<div class="wrapper">
+    <h2 class="header">Alex Corey</h2>
+    <nav class="nav__bar">
+        <a href="index.html" class="nav__bar__index nav__bar__item link">Home</a>
+        <a href="portfolio.html" class="nav__bar__portfolio nav__bar__item link">Portfolio</a>
+        <a href="resume.html" class="nav__bar__resume nav__bar__item link">Resume</a>
+    </nav>
+    <span class="picture__profile__container">
+        <img src="alex.jpg" class="profile__picture image" alt="Picture of Alex" />
+        <div class="about__me__all">
+            <h4 class="about__me__intro">Hello I am...</h4>
+            <h1 class="about_me_header main_header">Alex Corey</h1>
+            <h3 class="job_title">Software Developer</h3>
+            <p class="about__me__content">I am a software developer that works hard to solve provlems. Speklatr haussespekulant gaf stier aynasz specula la
```

Comment:
Please consider going back and filling in your placeholder text with real text: this project is meant to be your first step in building a portfolio.